database/migrations/2020_02_11_210851_add_description_column_to_club.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Please remove this file, for now we should have **one** migration file per table